pyannote/audio/pipelines/clustering.py

```diff
@@ -200,6 +232,19 @@ def __init__(self, metric: str = "cosine", expects_num_clusters: bool = False):
             ["Affinity", "Unnormalized", "RandomWalk", "GraphCut"]
         )
         self.eigengap = Categorical(["Ratio", "NormalizedDiff"])
+        self.spectral_min_embeddings = LogUniform(1, 100)
+
+        # Hyperparameters for refinement operations.
+        self.refinement_sequence = Categorical(
```

Comment:
Based on your experience, are there some combinations which are more promising than others? Once again, I'd rather avoid enlarging the search space too much... or pyannote.pipeline (based on optuna) will have lots of trouble converging towards the optimal set of hyperparameters.

Reply:
That's a very good point that we need to make the search space for optuna smaller.

Changed to `Categorical( ["O", "TS", "GTS", "CGTSDN"])`. These are the ones we experimented with before.